### DISCUSSION TITLE
[ponzi Alert] TMON project  ((The project was scammed - the end of project))

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#8] [General]
این سایت بر این  ادعاست که با سایت هایی مثل امازون همکاری میکنه بدینصورت که برای محصولات بازدید میزنه تا بازدید محصول بره بالاتر و راحتتر فروش رود.

توضیحات یکی از اعضای فعال :::

همانطور که در اینستاگرام ، توییتر ، یوتیوب و گوگل شما هر چه تعداد بازدید کننده و لایک کننده بیشتری داشته باشید اون پست شما در معرض دید همگان قرار می‌گیرد این شرکت هم از همین روش برای درآمدزایی و فروش محصولات خودش و طرفین قراردادش استفاده میکند.
شرکت کره ای بنام TMON  یک فروشگاه بین المللی بزرگ هست  که از سال 2009 میلادی شروع بکار کرده و از سال 2022  امسال از برج 3 میلادی بحث تبلیغات کلیکی رو راه انداری نموده است.
در جدول ثبت شرکت تاریخ 2031 را جهت ادامه ی روند کاری شرکت اعلام کرده اند.
که البته این یک اعلام ثبتی اجباری است و امیدواریم که بیش از این تاریخ شرکت فعالیت خود را ادامه دهد.
*پس این تاریخ شروع عالیه به این دلیل که اول راهیم...*
طرف قرارداد tmon شرکت‌هایی مثل Fnac، Allegro ، fruugo , wish , Amazon,  Zealando ,  هست .
با استفاده از تبلیغات کلیکی ، فروش محصولات این شرکت ها را بالا میبرد و باعث میشود محصولات این شرکت ها بازدید و لایک بیشتری بخورد و در قبال آن کلیکی که شما میکنید بستگی به میزان سرمایه شما سود روزانه پرداخت می‌کند.
 بابت هر ۱۰۰ دلار بصورت روزانه مبلغ ۴ دلار پرداخت می‌کند و شما ۵۰ کلیک رو میتوانید در ۱۵ دقیقه پشت سر هم بزنید.
مثال :
۱۰۰ $ = روزانه ۴ $ 
*که البته این ۴ درصد ثابت نمانده و به فرمول زیر در افزایش صعودی میباشد*
* ؟؟؟؟ صدم درصد سود روزانه + تعداد ؟؟؟؟ کلیک روزانه در گرب که برای مثال در آلگرو 26 صدم درصد است و 50 کلیک روزانه در گرب و در هر مرحله این سود دهی و تعداد کلیک متفاوت است....*
۲۰۰ $ = روزانه ۸ $  
۵۰۰ $ = روزانه ۲۰ $ 
۱۰۰۰ $ = روزانه ۴۰ $
۲۰۰۰ $ = روزانه ۸۰ $
۵۰۰۰ $ = روزانه ۲۰۰ $
حداقل برداشت شما ۳۰ دلار میباشد.
ضمنا شما هر زمان بخاهید میتوانید اصل سرمایه خودتان را برداشت کنید حتی یکساعت بعد از اینکه واریز کردید.
تمام واریز ها و برداشت ها بصورت دلار تتر trc20 میباشد .
شما کد دعوت هم دارید که در صورت تمایل میتوانید دوستان خود را با لینک معرفی خودتان وارد سیستم کنید و از مزایای آن هم برخوردار شوید.
صحت و درستی ادعای های مطرح شده مشخص نیست .

- توضیحات مطرح شده در مورد نحوه فعالیت در خود سایت : 
- - 
سیستم وابسته ما دارای 10 اتاق است که می تواند به عنوان 9 سطح نیز شناخته شود. پس از ثبت نام موفقیت آمیز، سطح پیش فرض اعضا سطح 1 (Allegro) است. وقتی حساب شما واریز شد، می توانید با گرفتن سفارش شروع به دریافت پورسانت کنید. برای باز کردن قفل به سطح 2 (Fruugo)، باید حداقل 5 خط مستقیم مستقیم معتبر و حداقل 500 دلار موجودی حساب در حساب خود داشته باشید. برای باز کردن قفل به سطح 3 (آمازون)، باید حداقل 12 خط مستقیم مستقیم معتبر و حداقل 2000 دلار موجودی حساب در حساب خود داشته باشید. برای باز کردن قفل به سطح 4 (Wish)، باید حداقل 20 خط مستقیم مستقیم معتبر و حداقل 5000 دلار موجودی حساب در حساب خود داشته باشید. برای باز کردن قفل به سطح 5 (سفیر Ⅰ)، باید حداقل 50 خط مستقیم مستقیم معتبر و حداقل 29999 دلار موجودی حساب در حساب خود داشته باشید. برای باز کردن قفل به سطح 6 (سفیر Ⅱ)، باید حداقل 80 خط مستقیم مستقیم معتبر و حداقل 79999 دلار موجودی حساب در حساب خود داشته باشید.
توجه: داون لاین دایرک معتبر به داون لاین سطح اول شما اطلاق می شود که مجموعاً 100 دلار واریز شده و 50 سفارش را در هر روز تکمیل کرده است. (ثبت نام شخصی یا حساب جعلی محاسبه نمی شود)
توجه کنید: آدرس واریز پلتفرم هر چند وقت یکبار به روز می شود. لطفاً هر زمان که واریز می کنید به صفحه سپرده USDT بروید تا آخرین آدرس گیرنده را دریافت کنید. اگر سوالی دارید، لطفا با خدمات مشتریان آنلاین ما تماس بگیرید.
هر مبلغ سفارش ارسال شده بین 10% تا 50% موجودی حساب کاربری است. هر عضوی که قادر به انجام 50-80 سفارش در روز است، به سطح/اتاق فعلی آنها بستگی دارد. به عنوان مثال: اگر وضعیت فعلی عضو سطح 4 (Wish) باشد، به این معنی که او حداقل 20 خط مستقیم مستقیم و حداقل 5000 دلار موجودی حساب دارد. او قادر به انجام 70 سفارش روزانه در روز خواهد بود و مبلغ سفارش تکی ارسالی بین 500 تا 2500 تومان خواهد بود و پورسانت این سفارش 1.6 تا 8 تومان (نرخ 0.32 درصد برای هر سفارش) خواهد بود.
(توجه: مقدار سفارش هر محصول یکسان نیست، به صورت تصادفی با توجه به نیاز سطح خود ارسال کنید
در همین حال، هر عضو می‌تواند درصدی از هر کمیسیون درآمد پایین‌دستی خود را مستقیماً از سامانه دریافت کند. بنابراین، هر چه یک عضو خطوط پایین‌تری داشته باشد، درآمد بیشتری از جوایز و کمیسیون‌ها به دست خواهد آورد. (لطفاً برای جزئیات بیشتر به صفحه "پاداش تبلیغاتی" مراجعه کنید). یادآوری دوستانه: سیستم یک حالت اعزام خودکار است. به منظور جلوگیری از تاخیر در شبکه و سایر خطاهای ارتباطی، لطفاً قبل از خروج از صفحه سفارش، پرداخت را تکمیل کنید. اگر پس از ارسال موفقیت آمیز محصول، سفارش توسط کاربر تایید و ارسال نشود، ممکن است نتواند به سفارشات ادامه دهد. برای ادامه کار، عضو باید دستور قبلی را پردازش کند.

تحقیقات :: 

ادرس سایت tmon-earnings.com و tmon11.com  در بررسی ها به سه ادرس رسیدم که احتمالا دامنه های بیشتری دارد.
سازندگان و افراد پشت پروژه نامشخص هستند .
در بررسیی whois  سایت هویت سازنده مخفی شده است .
راه اندازی سایت از تاریخ 12 نوامبر 2021 بوده است 
-	ادرس سایت tm8569.com یکی دیگر از دامنه هایی است که به همان سایت های بالایی متصل میشود و در تاریخ 16 مارچ 2022 ثبت شده است
باهرسه دامنه وارد یک سایت میشویدو میتوانید به اکانت خود وارد شوید.
برای ثبت نام در سایت حتما باید کد معرف داشته باشید و اختیاری نیست . این یعنی کل این کار بر پایه شبکه سازی اجباری است و این حرکت در شبکه های پانزی استفاده میشود مخصوصا اینکه برای شروع به کار ابتدا باید سرمایه گذاری کنید .

*بررسی ادرس سایت در scam adviser
![1](https://user-images.githubusercontent.com/103103225/176994470-692363bc-d3de-45d2-aca1-a349f7ffae83.png)

مشکلات سایت : 
هویت صاحب وب سایت در WHOIS پنهان است
رتبه ترانکو نسبتاً پایین است
وب سایت در یک کشور پرخطر واقع شده است.
سرور سایت دارای چندین وب سایت دیگر با بازبینی کم است
این وب سایت در حالی که بسیار جوان است، نظرات زیادی دارد
سن این سایت (بسیار) جوان است.

طبق ادعای افراد فعال ، زیرمجموعه شرکت tmon کره جنوبی است که فروشگاه کالاست و به ادرس https://www.tmon.co.kr/ است اما همچین ادعایی کذب است و حتی اطلاعات whois  این سایت کاملا مشخص است و با سایت ادعا شده متفاوت است .

+ فروشگاه تیمون با ادرس سایت tmon.co.kr
تاسیس می 2010
دفتر مرکزی : منطقه گانگنام سئول کره جنوبی
بنیانگذاران : دانیل شین، کیهیون کوان ، تامی دونگیون کیم ، کریستوفرسین
 
+ سایت تبلیغات کلیکی tmon-earnings.com - tmon11.com - tm8569.com
سازندگان و افراد پشت پروژه نامشخص هستند .
در بررسیی whois  سایت هویت سازنده مخفی شده است .
راه اندازی سایت از تاریخ 12 نوامبر 2021 ,  16 مارچ 2022 بوده است .
درخواست دهنده دو دامنه اول در ایالت اریزونای امریکا بوده است.
هشدار پانزی و کلاهبرداری

**Comment by micro73** [comment]
tm1235.com
tmon11.com
 tmon-earnings.com
tm8569.com
tmon-system.com

ادرس های شناسایی شده به شرح بالاست

**Comment by micro73** [comment]
![2022-07-04_11-02-54](https://user-images.githubusercontent.com/103103225/177098109-7d62ef3f-b698-43ea-9d87-3b961730aa84.jpg)

در رابطه با سایت های tmon کلاهبرداری به ادرس های کامنت قبل، به سایت اصلی (tmon.co.kr) ایمیل زدم و این جوابیه اونهاست :::
ترجمه ::

سلام،
این تیمون است.

از نامه ارزشمند شما متشکرم

تیمون ما یک وب سایت است که به خرید آنلاین در کره اختصاص داده شده است.

سایتی که درخواست کردید در کره قابل دسترسی نیست
قابل تایید نیست و من می خواهم یک بار دیگر به اطلاع شما برسانم که تیمون سایت جداگانه ای ندارد.

همچنین، لطفاً توجه داشته باشید که سایت های خارج از کشور به طور جداگانه اداره نمی شوند زیرا منحصراً در کره فعالیت می کنند.

لطفا توجه داشته باشید که حتی اگر یک لینک جداگانه برای من ارسال کنید، قابل بررسی و پردازش نیست.

از اینکه با محبت و علاقه با تیمون تماس گرفتید متشکرم.

متشکرم.

**Comment by Blockunity** [comment]
طی بررسی های انجام شده،
یکی از مجموعه دار های این پروژه که داخل ایران فعالیت داره و هر روزه تبلیغ کننده هست اینگونه پاسخ دادند:
![Screenshot_20220728-223020_Instagram.jpg](https://user-images.githubusercontent.com/72666334/181745638-bfe7b258-5538-4ade-bf87-20b1d8c6eb26.jpg)![Screenshot_20220728-223602_Instagram.jpg](https://user-images.githubusercontent.com/72666334/181745651-4e5c041b-05a8-44ff-9bb3-83b8e48b7a52.jpg)![Screenshot_20220728-223149_Instagram.jpg](https://user-images.githubusercontent.com/72666334/181745665-38d8377c-3764-424b-906a-141b5d68f1ca.jpg)![Screenshot_20220728-223131_Instagram.jpg](https://user-images.githubusercontent.com/72666334/181745681-44dfe7e7-30e9-426b-9c6e-d45965f574db.jpg)

صفحات اجتماعی این فرد.

**Comment by micro73** [comment]
![2022-08-03_10-28-20](https://user-images.githubusercontent.com/103103225/182535208-c09a299d-df86-4976-8bc7-8236a89bf39c.jpg)

پایان کار تیمون 
سایت هاش بسته شد .
